Frameworks/GLKit/GLKEffect.mm

```diff
@@ -87,7 +87,7 @@ - (BOOL)prepareShaders {
 }
 
 - (void)prepareToDraw {
-    BOOL success = FALSE;
+    bool success = false;
```

Comment:
nit: this should be isInvertible/canInvert etc to reflect the actual return.
I initially thought it was a indicator of pass/fail.

Also it seems this value is not being used, if we don't need it, we can just pass null in.